teacher.rb

```diff
@@ -1,4 +1,5 @@
 require './person'
+
 class Teacher < Person
   def initalize(age, specialization, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
```

Comment:
- [ ] When setting the classroom for a student it also adds it to the classrooms' students:

The class should be `class Student < Person` not `class Teacher < Person`, kindly implement that class.

Reply:
kindly check the students.rb

Reply:
This issue is well resolved! ✔️